ansible/roles/distributed_press/templates/nginx-static.j2

```diff
@@ -12,26 +12,36 @@ server {
 
   error_page 404 /404.html;
 
+  add_header Cache-Control no-cache;
   add_header Strict-Transport-Security "max-age=31536000; includeSubDomains; preload" always;
-  add_header 'X-Frame-Options' 'ALLOW-FROM *';
+  add_header X-Frame-Options "sameorigin";
+  add_header X-XSS-Protection "1; mode=block";
+  add_header X-Content-Type-Options "nosniff";
+  add_header Referrer-Policy "strict-origin-when-cross-origin";
```

Comment:
Referrer could be good for our analytics when users navigate to DP from the thing in the footer, is there a way to support that still?

Reply:
This will instruct browsers to set the referer only to the origin if the destination is secure, so third party websites can know from where the visit came from but not the particular page